app/views/shared/_run_table.slim

```diff
@@ -5,38 +5,34 @@
   center
     i No runs matched!
 - else
-  .card
-    - if description.present?
-      .card-header.text-white = description
+  - if description.present?
+    .card-header.text-white = description
+  .table-responsive
```

Comment:
Is there a reason to wrap everything below in the table-responsive instead of adding it to the table html element below?

Reply:
Good to know.  For some reason I thought in the past it was just attached to a table element.  Maybe in some last version it was like that or I'm just crazy.